README.md

```diff
@@ -2,21 +2,19 @@
 
 # Leaderboard
 
-> This projects is made to practice the use of APIs and ES6
+> This projects is made to practice the use of APIs and ES6 modules
 
-![screenshot](./app_screenshot.png)
+![screenshot](./src/assets/screenshot.jpg)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS & JavaScript
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://lino09.github.io/leaderboard/dist)
 
```

Comment:
- [ ] OPTIONAL
Kindly change the  heading title `Authors` to `Author` [ on line 40 of the ReadMe], so as to make your ReadMe professional

Reply:
Thanks for your help! I made the changes